compiler/src/dotty/tools/dotc/cc/SepCheck.scala

```diff
@@ -117,10 +117,10 @@ object SepCheck:
       if i < size then locs(i) else null
 
     def clashing(ref: Capability)(using Context): SrcPos | Null =
-      val refPeaks = ref.peaks
-      if !peaks.sharedWith(refPeaks).isEmpty then
+      val refPeaks = ref.directPeaks
+      if !directPeaks .sharedPeaks(refPeaks).isEmpty then
         var i = 0
-        while i < size && refs(i).peaks.sharedWith(refPeaks).isEmpty do
+        while i < size && refs(i).directPeaks .sharedPeaks(refPeaks).isEmpty do
```

Comment:
It seems that there is an extra `<space>` after every occurrence of `directPeaks`.

Reply:
Yes, I think it was a rename error.